pilot/pkg/bootstrap/server.go

```diff
@@ -877,6 +876,12 @@ func (s *Server) initRegistryEventHandlers() {
 				}: {}},
 				Reason: []model.TriggerReason{model.ConfigUpdate},
 			}
+			if curr.Namespace == "istio-system" && curr.Name == "default" && curr.GroupVersionKind == gvk.PeerAuthentication {
```

Comment:
Will this work if the Istio is installed in another namespace or it only applies to the `istio-system` namespace?

Reply:
This was just to get debug logs for a specific test scenario, won't merge with it.